### DISCUSSION TITLE
사용자 일괄 삭제시에 진행이 멈춥니다.

### DISCUSSION THREAD

**Discussion by KUBONKIL** [bmrdevteam/Altsis#289] [질문과 답변]
![image](https://github.com/bmrdevteam/school-information-system/assets/143776178/108bece6-e698-4d6d-af06-16666dab9996)

**Comment by KUBONKIL** [comment]
3명씩만 삭제됩니다.

**Comment by jessie129j** [comment]
해당 버그가 수정되었습니다! ([v1.9.14](https://github.com/bmrdevteam/school-information-system/releases/tag/v1.9.14))

**Reply by KUBONKIL** [reply]
빠른 수정 감사~
